br/pkg/lightning/config/config.go

```diff
@@ -143,6 +144,8 @@ type Config struct {
 	Cron         Cron                `toml:"cron" json:"cron"`
 	Routes       []*router.TableRule `toml:"routes" json:"routes"`
 	Security     Security            `toml:"security" json:"security"`
+	CheckOnlyCfg *CheckOnlyConfig    `toml:"-" json:"-"`
+	LogCfg       log.Config          `toml:"-" json:"-"`
```

Comment:
I prefer not to add a non-configurable filed only for internal usage here. If you really need this, please add it to `Lightning` and keep it consist with `GlobalLightning`.

Reply:
not good now , it's not good to pass it from `lightning` to `controller` too.

Reply:
I mean `Config.App` not the `Lightning` struct.

Reply:
mark (this conversation still need to discuss